lib/pengine/status.c

```diff
@@ -110,7 +110,7 @@ cluster_status(pe_working_set_t * data_set)
    if (!pcmk_any_flags_set(data_set->flags,
                            pcmk_sched_location_only|pcmk_sched_quorate)
        && (data_set->no_quorum_policy != pcmk_no_quorum_ignore)) {
-        crm_warn("Fencing and resource management disabled due to lack of quorum");
+        pcmk__config_warn("Fencing and resource management disabled due to lack of quorum");
```

Comment:
Let's keep this as crm_warn(), this doesn't need to be in the crm_verify output either